src/ratingsAndReviews/components/ratingsModalTable.jsx

```diff
@@ -0,0 +1,156 @@
+            <p>Poor</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row4" value="2.000" onChange={this.handleQualityChange.bind(this)} />
+            <p>Below average</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row4" value="3.000" onChange={this.handleQualityChange.bind(this)} />
+            <p>What I expected</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row4" value="4.000" onChange={this.handleQualityChange.bind(this)} />
+            <p>Pretty great</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row4" value="5.000" onChange={this.handleQualityChange.bind(this)} />
+            <p>Perfect</p>
+          </td>
+        </tr>
+        <tr id="rar_radioTableRow">
+          <td>Length</td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row5" value="1.000" onChange={this.handleLengthChange.bind(this)} />
+            <p>Runs short</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row5" value="2.000" onChange={this.handleLengthChange.bind(this)} />
+            <p>Runs slightly short</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row5" value="3.000" onChange={this.handleLengthChange.bind(this)} />
+            <p>Perfect</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row5" value="4.000" onChange={this.handleLengthChange.bind(this)} />
+            <p>Runs slightly long</p>
+          </td>
+          <td>
+            <input type="radio" id="radioTableSelector" name="rar_row5" value="5.000" onChange={this.handleLengthChange.bind(this)} />
+            <p>Runs long</p>
+          </td>
+        </tr>
+        <tr id="rar_radioTableRow6">
+          <td>Fit</td>
+          <td id="rar_td_header">
+            <input type="radio" id="radioTableSelector" name="rar_row6" value="1.000" onChange={this.handleFitChange.bind(this)} />
+            <p>Runs tight</p>
+          </td>
+          <td id="rar_td_header">
+            <input type="radio" id="radioTableSelector" name="rar_row6" value="2.000" onChange={this.handleFitChange.bind(this)} />
+            <p>Runs slightly tight</p>
+          </td>
+          <td id="rar_td_header">
+            <input type="radio" id="radioTableSelector" name="rar_row6" value="3.000" onChange={this.handleFitChange.bind(this)} />
+            <p>Perfect</p>
+          </td>
+          <td id="rar_td_header">
+            <input type="radio" id="radioTableSelector" name="rar_row6" value="4.000" onChange={this.handleFitChange.bind(this)} />
+            <p>Runs slightly long</p>
```

Comment:
Are these lines here where they are rating Fit supposed to match row 6 where they are rating Length?

Reply:
Great catch! That was a mistake from early styling experimentation I was doing.

Reply:
I have fixed and re-uploaded